### PR TITLE
Improve incident_attributes.rst grammar

### PR DESCRIPTION
**Pull request by stavares843**
Improve incident_attributes.rst grammar

### CODE REVIEW

**Comment by stavares843** [comment]
cc @nickrohrbs

File: docs/incident_attributes.rst [hunk 1]
```diff
@@ -3,9 +3,9 @@
 Incidents and event attributes
 ********************************
 
-An incident is an alert that gets fired. This groups together any number of events that are worth alerting on. A single SSH Login Attempt incident can be made up of multiple attempts to use different username and password combinations against a Canary. Likewise a single HTTP Login Attempt incident, can be made up of multiple HTTP POST events to a Canary website. The events are bundled together if they occur within a short time of each other and from a single attacker. Brute force attempts on a service then gets grouped together into an incident.
+An incident is an alert that gets fired. This groups together any number of events that are worth alerting on. A single SSH Login Attempt incident can be made up of multiple attempts to use a different username and password combinations against a Canary. Likewise, a single HTTP Login Attempt incident can be made up of multiple HTTP POST events to a Canary website. The events are bundled together if they occur within a short time of each other and from a single attacker. Brute force attempts on a service then get grouped together into an incident.
```
Comment (nickrohrbs) [inline docs/incident_attributes.rst:6]:
```suggestion
An incident is an alert that gets fired. This groups together any number of events that are worth alerting on. A single SSH Login Attempt incident can be made up of multiple attempts that use different username and password combinations against a Canary. Likewise, a single HTTP Login Attempt incident can be made up of multiple HTTP POST events to a Canary website. The events are bundled together if they occur within a short time of each other and from a single attacker. Brute force attempts on a service then get grouped together into an incident.
```

File: docs/incident_attributes.rst [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
 
-Each incident object has an **events** attribute storing its list of :class:`Event <Event>` objects. This page describes the different attributes events have depending on their type. Each event also contains an "updated_std" timestamp field, which is omitted for brevity below.
+Each incident object has an **events** attribute storing its list of :class:`Event <Event>` objects. This page describes the different attributes events have depending on their type. Each event also contains a "updated_std" timestamp field, which is omitted for brevity below.
```
Comment (nickrohrbs) [inline docs/incident_attributes.rst:8]: "an" seems correct here?

File: docs/incident_attributes.rst [hunk 3]
```diff
@@ -292,10 +292,10 @@ The :class:`Event <Event>` object in this scenario will have the following attri
 
 Port Scans
 ==========
-There are five types of port scans incidents.
+There are five types of port scan incidents.
 
-#. A host port scan occurs when a single Canary is port scanned by a single source.
-#. A consolidated network port scan occurs when multiple Canaries are scanned by a single source.
+#. A host port scan occurs when a single Canary is a port scanned by a single source.
```
Comment (nickrohrbs) [inline docs/incident_attributes.rst:297]: This change ("a") isn't right, we should undo the change.

File: docs/incident_attributes.rst [hunk 4]
```diff
@@ -292,10 +292,10 @@ The :class:`Event <Event>` object in this scenario will have the following attri
...
+#. A consolidated network port scan occurs when the multiple Canaries are scanned by a single source.
```
Comment (nickrohrbs) [inline docs/incident_attributes.rst:298]: This change should also be undone, adding "the " is wrong.

File: docs/incident_attributes.rst [hunk 5]
```diff
@@ -360,7 +360,7 @@ The :class:`Event <Event>` object in this scenario will have the following attri
 
 Canary Disconnected
 ======================
-Event is generated when a Canary does not contact the console within a defined time period.
+An event is generated when a Canary does not contact the console within a defined time period.
```
Comment (nickrohrbs) [inline docs/incident_attributes.rst:363]: Either leave it as "Event is .." (it's just a description of the event) or change it to "A Disconnect event is... "

File: docs/incident_attributes.rst [hunk 6]
```diff
@@ -457,7 +457,7 @@ Microsoft SQL Server Login Attempt
 ===================================
 Triggered by any attempt to authenticate to the Microsoft SQL Server module.
 
-SQL Server supports multiple authentication modes, and the fields that come through depend on the mode.
+SQL Server supports multiple authentication modes and the fields that come through depending on the mode.
```
Comment (nickrohrbs) [inline docs/incident_attributes.rst:460]: We are talking about the fields that are returned in the event, and they "depend" (not "depending") on the mode.

**Review by nickrohrbs — CHANGES_REQUESTED** [review]

File: docs/incident_attributes.rst (hunk 4 above)
Comment (stavares843) [reply docs/incident_attributes.rst]: updated 🔨

File: docs/incident_attributes.rst (hunk 3 above)
Comment (stavares843) [reply docs/incident_attributes.rst:297]: updated 🔨

File: docs/incident_attributes.rst (hunk 2 above)
Comment (stavares843) [reply docs/incident_attributes.rst]: updated 🔨

**Comment by stavares843** [comment]
updated @nickrohrbs 

all these were grammarly suggestions, reverted what you asked 😂
